### PR TITLE
Geomap: fix broken demo layers

### LINKED ISSUE #9068: [BUG] Geomap - Sample layers no longer available
On the plugin sample page, the following layers are no longer accessible:

* WMS
* JSON (Earthquakes)
* EsriJSON

We need to find new service endpoints for these samples.

Please assign me to this bug/task.

### PR DESCRIPTION
**Pull request by jvanulde**
This PR fixes the broken demo layers. It also adds a new layer format (Esri REST Tile). A long-standing but unreported bug with the JSON path configuration has been fixed. Minor updates to the plugin documentation.

Fixes #9068

### CODE REVIEW

File: src/plugins/geomap/demo/config-fr.js [hunk 1]
```diff
@@ -93,7 +102,7 @@ var wet_boew_geomap = {
 						"oKS8uknovhu0LxRwllpRFyQDnTMta43C4fp0qb9rxVlTXexNY9S1Lb5rhp4Ee07LSpkMiQhNYrRCDzZJks0XKEyI45Ha7RaScGT9YRXFHd5nNEH8C+eGD9m6tNTgAAAAASUVORK5CYII=",
 						fillOpacity: "1",
 						graphicWidth: "25",
-						name: "Mines de charbon"
+						name: "Coal mines"
```
Comment (duboisp) [inline src/plugins/geomap/demo/config-fr.js:105]: Do we need to use the English name? Can we keep the French name?

File: src/plugins/geomap/demo/config-fr.js [hunk 2]
```diff
@@ -108,7 +117,7 @@ var wet_boew_geomap = {
 						"p91tLJi/b3ppudc1jH2z3b1vhp4Ehx3WrlkOKQvNYp5STfUKoi1/Jv4WUualAICCL5ax4YTUtMLFSzmriL1z5gWQ67XYNAAAAAElFTkSuQmCC",
 						fillOpacity: "1",
 						graphicWidth: "25",
-						name: "Mines de métaux"
+						name: "Metal mines"
```
Comment (duboisp) [inline src/plugins/geomap/demo/config-fr.js:120]: Do we need to use the English name? Can we keep the French name?

File: src/plugins/geomap/demo/config-fr.js [hunk 3]
```diff
@@ -123,7 +132,7 @@ var wet_boew_geomap = {
 						"7yU8DvztaN22EZIXSWIdSf30jRE5OtjylklG/36+y5Wx6YaX0Z16e24i/AWg1hRJKTWppAAAAAElFTkSuQmCC",
 						fillOpacity: "1",
 						graphicWidth: "25",
-						name: "Mines de non-métaux"
+						name: "Nonmetal Mines"
```
Comment (duboisp) [inline src/plugins/geomap/demo/config-fr.js:135]: Do we need to use the English name? Can we keep the French name?

File: src/plugins/geomap/demo/config-fr.js [hunk 4]
```diff
@@ -138,7 +147,7 @@ var wet_boew_geomap = {
 						"AgGXbm1wuz7VF85YqIHXPsTjW6lu3dpmfBvot2y5ZCkkJxbE2pW6cFCItLVWeUrEpr9erUuUse2Gl9CZfnrsYfwFqWYQCxROHDQAAAABJRU5ErkJggg==",
 						fillOpacity: "1",
 						graphicWidth: "25",
-						name: "Mines de sables bitumineux"
+						name: "Oil sands mines"
```
Comment (duboisp) [inline src/plugins/geomap/demo/config-fr.js:150]: Do we need to use the English name? Can we keep the French name?

**Review by duboisp — CHANGES_REQUESTED** [review]
I am fine with the changes except we should keep the French translation.

Todo: Test locally to ensure it is working

File: src/plugins/geomap/demo/config-fr.js (hunk 1 above)
Comment (jvanulde) [reply src/plugins/geomap/demo/config-fr.js]: Fixed in 27caeb5aa2b3fef1718e261c90ebec24dba2c448

File: src/plugins/geomap/demo/config-fr.js (hunk 2 above)
Comment (jvanulde) [reply src/plugins/geomap/demo/config-fr.js]: Fixed in 27caeb5aa2b3fef1718e261c90ebec24dba2c448

File: src/plugins/geomap/demo/config-fr.js (hunk 3 above)
Comment (jvanulde) [reply src/plugins/geomap/demo/config-fr.js]: Fixed in 27caeb5aa2b3fef1718e261c90ebec24dba2c448

File: src/plugins/geomap/demo/config-fr.js (hunk 4 above)
Comment (jvanulde) [reply src/plugins/geomap/demo/config-fr.js]: Fixed in 27caeb5aa2b3fef1718e261c90ebec24dba2c448

File: src/plugins/geomap/demo/config-fr.js [hunk 5]
```diff
@@ -281,35 +307,35 @@ var wet_boew_geomap = {
...
+			caption: "Ceci est un exemple de jeu de données chargé à partir d'une ressource GeoJSON distante, dans ce cas les caméras de circulation dans la ville d'Ottawa à partir de l'API CartoDB",
 			type: "geojson",
 			url: "//stephenott.cartodb.com/api/v2/sql",
 			params: {
 				format: "GeoJSON",
 				q: "SELECT * FROM traffic_cameras LIMIT 25"
 			},
 			attributes: {
-				location_desc: "Emplacement",
+				location_desc: "Location",
```
Comment (duboisp) [inline src/plugins/geomap/demo/config-fr.js:318]: Can we keep the French term "Emplacement"?

File: src/plugins/geomap/demo/config-fr.js [hunk 6]
```diff
@@ -281,35 +307,35 @@ var wet_boew_geomap = {
...
 				longitude: "Latitude",
 				latitude: "Longitude",
-				updated_at: "Dernière mise à jour"
+				updated_at: "Last updated"
```
Comment (duboisp) [inline src/plugins/geomap/demo/config-fr.js:321]: Can we keep the French term "Dernière mise à jour"?

File: src/plugins/geomap/demo/config-fr.js [hunk 7]
```diff
@@ -154,45 +163,62 @@ var wet_boew_geomap = {
...
 			visible: false,
 			popups: true,
 			datatable: true,
 			zoom: true,
+			root: "features",
 			attributes: {
-				location: { path: "en", alias: "Localisation" },
-				magnitude: "Magnitude",
-				origin_time: "Time"
+				title: { path: "properties", alias: "Title" },
```
Comment (duboisp) [inline src/plugins/geomap/demo/config-fr.js:219]: The French for "title" is "Titre"

File: src/plugins/geomap/demo/config-fr.js [hunk 8]
```diff
@@ -154,45 +163,62 @@ var wet_boew_geomap = {
...
+				mag: { path: "properties", alias: "Magnitude" },
+				time: { path: "properties", alias: "Time" }
```
Comment (duboisp) [inline src/plugins/geomap/demo/config-fr.js:221]: The French for "Time" is "Temps"

File: src/plugins/geomap/demo/config-fr.js [hunk 9]
```diff
@@ -154,45 +163,62 @@ var wet_boew_geomap = {
...
 		{
 			title: "EsriJSON",
-			caption: "*NEEDS TRANSLATION*This is a sample dataset loaded from a remote Esri JSON resource.",
+			caption: "Ceci est un exemple de jeu de données chargé à partir d'une ressource Esri JSON distante.",
 			type: "esrijson",
-			url: "//geoappext.nrcan.gc.ca/arcgis/rest/services/NEB/Pipeline/MapServer/0/query?where=OBJECTID>0&f=pjson",
+			url: "https://geoappext.nrcan.gc.ca/arcgis/rest/services/FGP/TMX_EN/MapServer/2/query?where=OBJECTID>0&f=pjson",
 			attributes: {
-				OPERATOR: "Opérateur"
+				OPERATOR: "Operator"
```
Comment (duboisp) [inline src/plugins/geomap/demo/config-fr.js:198]: Can we keep the French term "Opérateur"?

**Review by duboisp — CHANGES_REQUESTED** [review]
@jvanulde There is still a few text to switch back to their French translations. Sorry I missed it during the last review.

Let me know if you don't have time to do those requested change, I may able to find someone to fix those small changes.

For record keeping: After reviewed, this PR add new feature to geomap plugin, so it will increase WET-BOEW by a minor version. 

Thanks

File: src/plugins/geomap/demo/config-fr.js (hunk 5 above)
Comment (jvanulde) [reply src/plugins/geomap/demo/config-fr.js]: Fixed

File: src/plugins/geomap/demo/config-fr.js (hunk 6 above)
Comment (jvanulde) [reply src/plugins/geomap/demo/config-fr.js]: Fixed

File: src/plugins/geomap/demo/config-fr.js (hunk 7 above)
Comment (jvanulde) [reply src/plugins/geomap/demo/config-fr.js]: Fixed

File: src/plugins/geomap/demo/config-fr.js (hunk 8 above)
Comment (jvanulde) [reply src/plugins/geomap/demo/config-fr.js]: Fixed

File: src/plugins/geomap/demo/config-fr.js (hunk 9 above)
Comment (jvanulde) [reply src/plugins/geomap/demo/config-fr.js]: Fixed

**Review by duboisp — APPROVED** [review]

**Pull request merged**